app/Http/Controllers/TemplateController.php

```diff
@@ -144,16 +148,23 @@ public function destroy(Request $request, Template $template)
         }
     }
 
-    public function changeStatus(Request $request, Template $template)
+    public function changeStatus(Request $request)
     {
+        $template = $this->templateRepository->find($request->id);
         $this->authorize('update', $template);
 
-        if ($request->status === TemplateStatus::STATUS_PUBLIC) {
+        if ($request->status == TemplateStatus::STATUS_PUBLIC) {
             $status = TemplateStatus::STATUS_PUBLIC;
-        } else {
+        }
+
+        else if ($request->status == TemplateStatus::STATUS_REVIEWING) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.ControlStructures.ElseIfDeclaration.NotAllowed</br>**Severity**: warning</br>**File**: app/Http/Controllers/TemplateController.php</br>**Line**: 160</br>Usage of ELSE IF is discouraged; use ELSEIF instead